src/main/java/streams/cta/io/ImageStream.java

```diff
@@ -26,13 +22,26 @@
     public ImageStream(SourceURL url) {
         super(url);
     }
-
     public ImageStream() {
     }
 
+
+    private class MC {
+        double energy, alt, az, coreY, coreX;
+    }
+    private class Array {
+        int[] triggeredTelescopes;
+        int numTriggeredTelescopes;
+    }
+    private class Event{
+        Map<Integer, double[]> images;
+        MC mc;
+        Array array;
+        long eventId;
+    }
+
```

Comment:
some minor comment about these private classes would be nice (although, most of the things are clear at the moment, but for the purity of the project!)